### DISCUSSION TITLE
Lenovo T580 Ventura 13.0.1

### DISCUSSION THREAD

**Discussion by humble-b** [valnoxy/t480-oc#53] [General]
Setup works with ventura 13.0.1, i needed to do a few thing though

Update opencore to latest version 0.8.6
Update all kexts to latest version
For wifi updated to Airporttlwm.kext Alpha 2

After this i was able to successfull boot into Ventura

What still doesn't work:
bluetooth 
dual screen setup. External connector isn't detected at all
Dual boot into windows

I also notice some acpi errors in Hackintool kenel logs

The dual screen is the most blocking for me.

**Comment by askwakhid** [comment]
Do you use USB-C port for external display ?
